methods/userEvents.coffee

```diff
@@ -1,27 +1,26 @@
 Incidents = require '/imports/collections/incidentReports.coffee'
 UserEvents = require '/imports/collections/userEvents.coffee'
+UserEventSchema = require '/imports/schemas/userEvent.coffee'
 
 Meteor.methods
-  editUserEvent: (id, name, summary, disease) ->
-    if Roles.userIsInRole(Meteor.userId(), ['admin'])
-      user = Meteor.user()
-      now = new Date()
-      trimmedName = name.trim()
-
-      if trimmedName.length
-        UserEvents.upsert id,
-          $set:
-            eventName: trimmedName
-            summary: summary
-            disease: disease
-            lastModifiedDate: now
-            lastModifiedByUserId: user._id
-            lastModifiedByUserName: user.profile.name
-          $setOnInsert:
-            creationDate: now
-            createdByUserId: user._id
-            createdByUserName: user.profile.name
-            articleCount: 0
+  upsertUserEvent: (userEvent) ->
```

Comment:
I noticed that before we were trimming name, before we upserted.  Just wanted to confirm that we no longer need to do this.